fixtures/js/render.ts

```diff
@@ -0,0 +1,5 @@
+function render() {
```

Comment:
What about renaming this to `render.tsx` - it would slightly enhance the test, since it would take advantage of the resolve.extensions change and make sure the loader applies to both extensions